### DISCUSSION TITLE
Minimap

### DISCUSSION THREAD

**Discussion by cycyx** [deckerst/aves#1124] [Q&A] [answered]
How is this overlay working?

If toggled in the viewer overlays, no minimap is ever displayed.

Is it relying on actual GPS data or can it use geolocation made in Aves?

**Comment by deckerst** [comment, marked as answer]
The minimap is not related to geographic coordinates. It's indicating the position of the view inside the image when you are zoomed in. It only appears when zoomed in.

If you want a map you can swipe up to show the Info page, which includes a small map preview.
